payloads/library/GitBunnyGit/README.md

```diff
@@ -5,15 +5,15 @@ Version: 1.0
 
 ## Description
 
-Clones the bashbunny-payloads repository and also will update an existing repository.
+Clones the bashbunny-payloads repository and also will update an existing repository. Use this payload to get all the other payloads!
 
 ## Configuration
 
 Configured for nix by default. Swap RNDIS_ETHERNET with ECM_ETHERNET on Windows
 
 ## Requirements
 
-Target must be sharing internet.
+Target must be sharing internet and have git installed.
```

Comment:
Ambiguous.  Does the BB need git, or the target providing the internet sharing connection?
Suggest:
`Target must be sharing internet.
BashBunny must have git installed.`